### PR TITLE
Newsweek Responsive - Bootstrap

### PR DESCRIPTION
**Pull request by gasb150**
In this project, we clone the Newsweek homepage using semantic HTML5 elements that are appropriately sized and positioned.
We use bootstrap to position and sizing the elements, and make the page responsive at the Bootstrap breakpoints.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,44 @@
...
+> In this project we did a responsive design of Newsweek Homepage.
+
+![screenshot](../abdo-branch/screenshot.png)
+
+In this project, we clone the Newsweek homepage using semantic HTML5 elements that are appropriately sized and positioned.
+We use bootstrap to position and sizing the elements, and make the page responsive at the Bootstrap breakpoints.
+
+## Live Demo 
+
+- [Visit Page] (https://raw.githack.com/AbdelrhmanAmin/Microverse-Bootstrap/feature-branch/index.html)
```
Comment (ioilmio) [inline README.md]: add an exclamation mark at the start of this line to use links in markdown

File: index.html [hunk 2]
```diff
@@ -0,0 +1,865 @@
...
+  <header class="container-fluid mx-1">
+    <nav class="navbar bg-danger fixed-top w-100 navbar-dark py-1 px-2 px-md-3 px-lg-5 py-lg-0 ">
+    <div class="container-fluid m-0 p-0 d-flex flex-row align-items-center">
+      <div class="row d-flex w-100 flex-nowrap">
+        <div class="d-none d-lg-flex flex-column justify-content-center col-lg-4 text-light">
+          <span><i class="fas fa-cloud-sun-rain"></i> 20° Alexandria, Egypt</span>
+          <span>Thu, Nov 12, 2020</span>
+        </div>
+        <a href="#" class="navbar-brand col-6 col-lg-4">
+          <h4 class="m-0 h4">Kittensweek</h4>
```
Comment (ioilmio) [inline index.html]: This should be the actual Newsweek logo

File: index.html [hunk 3]
```diff
@@ -0,0 +1,865 @@
...
+        </a>
+        <ul class="navbar-nav d-flex flex-row align-items-center justify-content-end col-6 col-lg-4">
+          <li class="d-none d-md-block">
+            <small class="m-0 text-light font-weight-bold text-uppercase">Login</small>
```
Comment (ioilmio) [inline index.html]: This should be a link

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,209 @@
+button {
+  line-height: initial !important;
```
Comment (ioilmio) [inline src/style.css]: Avoid !important rules

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,209 @@
...
+}
+
+.navbar-toggler-icon {
+  width: 1.1em !important;
```
Comment (ioilmio) [inline src/style.css]: Avoid !important rules

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,209 @@
...
+}
+
+.bg-danger,
+.btn-danger {
+  background-color: #f72210 !important;
```
Comment (ioilmio) [inline src/style.css:11]: Avoid !important rules

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,209 @@
...
+}
+
+.bg-warning,
+.btn-warning {
+  background-color: #ffe400 !important;
```
Comment (ioilmio) [inline src/style.css:16]: Avoid !important rules

File: src/style.css [hunk 8]
```diff
@@ -0,0 +1,209 @@
...
+}
+
+body {
+  background-color: #f4f5f7;
+  width: 100vw;
+  overflow-x: hidden;
+}
+
+.border-top-manual {
+  border-top: 5px solid #dee2e6 !important;
```
Comment (ioilmio) [inline src/style.css:35]: Avoid !important rules

File: src/style.css [hunk 9]
```diff
@@ -0,0 +1,209 @@
...
+}
+
+.border-bottom-double-manual {
+  border-bottom: 3px double #dee2e6;
+}
+
+.border-bottom-manual {
+  border-bottom: 5px solid #dee2e6 !important;
```
Comment (ioilmio) [inline src/style.css:43]: Avoid !important rules

File: src/style.css [hunk 10]
```diff
@@ -0,0 +1,209 @@
...
+.card-img {
+  width: 100%;
+  height: 250px !important;
+}
+
+.article span img,
+.img-icon {
+  width: 29px !important;
+  height: 28px !important;
+}
```
Comment (ioilmio) [inline src/style.css]: Avoid !important rules

File: src/style.css [hunk 11]
```diff
@@ -0,0 +1,209 @@
...
+
+.left-content,
+.right-content,
+.middle-content {
+  background: white;
+}
+
+.search-div {
+  width: 5%;
+  margin-left: 10px !important;
```
Comment (ioilmio) [inline src/style.css]: Avoid !important rules

File: src/style.css [hunk 12]
```diff
@@ -0,0 +1,209 @@
...
+.colored-space {
+  width: 233px;
+  height: 15px;
+  background-color: #f72210;
+  position: absolute;
+  right: 0;
+}
+
+.sm-img {
+  height: 200px !important;
```
Comment (ioilmio) [inline src/style.css]: Avoid !important rules

File: src/style.css [hunk 13]
```diff
@@ -0,0 +1,209 @@
...
+  }
+
+  .News {
+    height: 180px !important;
+  }
+
+  .sm-img {
+    height: 100% !important;
+  }
+}
```
Comment (ioilmio) [inline src/style.css]: Avoid !important rules

File: src/style.css [hunk 14]
```diff
@@ -0,0 +1,209 @@
...
+
+  .form-lg {
+    flex: 0 0 100% !important;
+    max-width: 100% !important;
+  }
+
+  #News {
+    height: 280px !important;
+  }
+}
```
Comment (ioilmio) [inline src/style.css]: Avoid !important rules

File: index.html [hunk 15]
```diff
@@ -0,0 +1,865 @@
...
+                  alt="kitty" />
+                <div class="d-flex flex-column justify-content-center">
+                  <p class="h6 font-weight-bolder text-capitalize">
+                    Why the cats are domestic? to submit humans to their masterplan?
+                  </p>
+                  <p class="small font-weight-bolder text-secondary">
+                    ASKS THE NEKO MASTER
+                  </p>
+                </div>
+              </div>
```
Comment (ioilmio) [inline index.html]: fix this take a look
image
)

File: index.html [hunk 16]
```diff
@@ -0,0 +1,865 @@
...
+
+  <div class="container-fluid">
+    <div class="row no-glutters mx-1">
+       <div class="col-12 bg-white">
+         <div class="article bg-white pr-3 pl-3 pt-3 pb-1">
+        <div class="d-flex justify-content-center align-items-center mb-3 border-bottom pb-3">
+          <span class="font-weight-bold">
+            <img class="mr-2" src="https://placekitten.com/g/200/300" alt="Magazine" /> IN
+            THE MAGAZINE
+          </span>
```
Comment (ioilmio) [inline index.html]: remember: your building a clone stick with the original design

image

File: index.html [hunk 17]
```diff
@@ -0,0 +1,865 @@
...
+       </div>
+      <!-- EDITORS PICKS -->
+      <div class="col-12 bg-white">
+        <div class="row">
+          <div class="article bg-white mb-3 mb-lg-0 pr-3 pl-3 pt-3 pb-1 border-bottom">
+        <div class="d-flex justify-content-center align-items-center mb-3 pb-3 border-bottom col-12">
+          <span class="font-weight-bold">
+            <img class="mr-2" src="https://placekitten.com/g/200/300" alt="editors pick" />
+            EDITOR'S PICK</span>
+        </div>
```
Comment (ioilmio) [inline index.html]: Same above, stick to the original design

**Review by ioilmio — CHANGES_REQUESTED** [review]
## Hello 👋🏼  @AbdelrhmanAmin and @gasb150 nice job on this project! 🎉 
## I love the cat idea but this is a clone project and we have to stick to the original design;
### There are some comments on your code, follow them please

So we have to make to fix a couple of issues, let's tart 👊🏼 

- It should contain main navigation links like in the original page [this ](https://pasteboard.co/JAflwI4.png);

- It should contain a Featured slideshow section like [this](https://pasteboard.co/JAfpI5L.png)

- Some images are squeezed and looks pretty bad like in this [case](https://pasteboard.co/JAfqC0c.png) please check for them in various screen sizes and fix them

- Refactor your footer:
   - It should contain Newsweek logo, social media icons and  main navigation links 

## Next time use GitHub Flow appropriately and switch roles more often
 Is not clear "who" has worked on "what";
### You can improve a bit your read me file if you include images [here](https://gist.github.com/vinkla/dca76249ba6b73c5dd66a4e986df4c8d)'s a guide on how to do that

File: index.html (hunk 15 above)
Comment (AbdelrhmanAmin) [reply index.html]: This is not appearing on my screen nor on my coding partner's screen!

**Comment by mimipeshy** [comment]
## STATUS: CHANGES REQUIRED :recycle: 

Hey team :wave: this is an amazing job you guys did, you implemented the changes required. However, we still  need to make a few more changes as below :smile: 

- [x] Update your readme screenshot image, it's broken.

- [x] images in the 'in the magazine section' should have an equal height and width, the current images look squeezed.

- [x] Make your commit messages more descriptive e.g 

> fix readme screenshot image

- [x] Remove the grey color that's after the 2nd menu the goal is to make it look like the original site. See below

**Original**

 Image of original

**clone**
p

That's all :relieved:  , now smile and submit another request when done.

Regards 
Peris Ndanu
Feel free to slack me if you have any questions

**Review by ebeagusamuel — APPROVED** [review]
# Project Approved :100: 
Hello :wave: 
Nice work implementing the changes requested by the previous TSE's.
You can go ahead now and merge the branch.
Good luck with the next project.
slack(@ebeagusamuel)

**Pull request merged**